README.md

```diff
@@ -6,3 +6,20 @@ This is a place for style configurations for [Rainforest QA](https://www.rainfor
 
 - Make sure you're an owner on rubygems.org
 - `rake release`
+
+
+## Adding `rf-stylez` to a new project
+
+Create a `.rubocop.yml` in the root project directory and paste the following:
+```yml
+inherit_gem:
+  rf-stylez: ruby/rubocop.yml
+Style/HashSyntax:
+  Enabled: true
+```
+
+To use it install rubocop and rf-stylez locally:
+```bash
+gem install rubocop
+gem install rf-stylez
```

Comment:
This is correct, but it seems brittle because `rubocop` tends to have breaking changes and renames cops at will (it's pre-1.0 still), so this is unfortunately not guaranteed to work unless you have a version of rubocop whose cop names match what is used in `rf-stylez` 😭

Reply:
If you install just `rf-stylez` it will install `rubocop` in a correct version given it's a dependency, so if we 🔥 line `23` we're good?

Also it's not too much effort to keep `rf-stylez` up to date with rubocop